src/Pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,12 @@
+import { Navigation, Footer } from "../../Components/UI/";
```

Comment:
Why you import these here?

Reply:
Forgot to remove it. Thanks for pointing it out.